test/e2e-go/features/devmode/devmode_test.go

```diff
@@ -50,18 +50,32 @@ func TestDevMode(t *testing.T) {
 	txn := fixture.SendMoneyAndWait(0, 100000, 1000, sender.Address, receiver.String(), "")
 	require.NotNil(t, txn.ConfirmedRound)
 	firstRound := *txn.ConfirmedRound + 1
-	start := time.Now()
+	blk, err := fixture.AlgodClient.Block(*txn.ConfirmedRound)
+	require.NoError(t, err)
+	seconds, _ := math.Modf(blk.Block["ts"].(float64))
+	startTime := time.Unix(int64(seconds), 0)
+	blkOffset := uint64(1_000_000)
```

Comment:
tiny nit: maybe set this as a constant (unfortunately still need to convert uint64 to a time.duration below though) and add a comment on testing consecutive timestamps in dev mode